### DISCUSSION TITLE
Keyword for Search Engine Recommendation

### DISCUSSION THREAD

**Discussion by hanyangclarence** [alshedivat/al-folio#2109] [Q&A] [answered]
Hi, I am a newbie in writing website. I'd like to know that whether I can set some keyword in my files, so that the search engine can better recommend my personal website. I've just created my website, and currently if I search my name and university abbreviation, the website is not shown.

If there is no such option, is there any other way to better recommend my website? 

Thank you!

**Comment by george-gca** [comment, marked as answer]
Yes, you can set them in [_config.yml](https://github.com/alshedivat/al-folio/blob/30f7ab0f11eae400a9bb4f19256054e197954a1f/_config.yml#L16C1-L16C9)

**Reply by hanyangclarence** [reply]
Ahh I see. Thank you a lot!
